Light.json

```diff
@@ -56989,6 +56999,8 @@
         "imageUrl": "https://res.starwarsccg.org/cards/VirtualPremium-Light/large/puck.gif",
         "type": "Character",
         "subType": "Rebel",
+        "uniqueness": "*",
+        "destiny": "7 or 1",
```

Comment:
Please use the high number of `7` as the destiny.
For the low number of `1`, please place in a field of `uniqueDestiny: "1`